setup.py

```diff
@@ -42,7 +42,11 @@
                     'pytz',
                     'six',
                     ]
-TESTS_REQUIRE = ['pytest', 'nose']
+TESTS_REQUIRE = ['pytest', 'pytest-cov', 'pytest-mock', 'nose']
+EXTRAS_REQUIRE = {
+    'complete': ['scipy', 'tables', 'numba', 'siphon', 'ephem'],
```

Comment:
please add `netcdf4`

Reply:
thanks for catching this. working on adding a ``doc`` section, too.